reproducibility_project/src/engine_input/mcccs/fort77maker_twobox.py

```diff
@@ -0,0 +1,457 @@
+"""Takes an mbuild filled_box and creates a restart file for mcccs simulations."""
+import errno
+import os
+import sys
+from collections import Counter
+from subprocess import call
+
+import mbuild as mb
+import numpy as np
+
+print(sys.executable)
+
+
+def unique(list1):
+    """Delete duplicate elements in a list.
+
+    Parameters
+    ----------
+    list1 : Input list that may contain duplicate elements
+
+    Returns
+    -------
+    unique_list : Output list with duplicate elements deleted
+
+    """
+    unique_list = []
+
+    # traverse for all elements
+    for x in list1:
+        # check if exists in unique_list or not
+        if x not in unique_list:
+            unique_list.append(x)
+    # print list
+    return unique_list
+
+
+# This function takes name_list, coordinates, and writes the xyz file in out_filename
+def xyzwriter(name_list, coordinates, out_filename):
```

Comment:
Why not use `mbuild.formats.xyz.write_xyz`?

Reply:
I made my own xyz write because mBuild only gives 6 decimal digits. I need at least 7 for my software because when the bonds are rigid, all bonds must be of the exact same length, with a precision of 1e-7 \AA.

Reply:
hmm. Maybe this should be a kwarg to our xyz writer.